test/with_qiskit/test_circuit_assert_equal.py

```diff
@@ -14,12 +14,12 @@ class TestCircuitAssertEqual(unittest.TestCase):
     How to execute this test:
     $ pwd
     {Your directory where you git-cloned quantestpy}/quantestpy
-    $ python -m unittest test.test_circuit_assert_equal
-    ...
+    $ python -m unittest test.with_qiskit.test_circuit_assert_equal
+    .s..........
     ----------------------------------------------------------------------
-    Ran 3 tests in 0.006s
+    Ran 12 tests in 0.012s
 
-    OK
+    OK (skipped=1)
```

Comment:
Doesn't `skipped=1` cause trouble?

Reply:
Thanks for this comment. One test is skipped on purpose (the reason was that rx or rz gate was not implemented at that time, or something like that). 
I will check it.

Reply:
@mitz1012 
> Doesn't `skipped=1` cause trouble?

I deleted this test because I have implemented this test without understanding correctly 👉 https://github.com/QuantestPy/quantestpy/pull/124/commits/22dfd4c16a5b5e915611fc7ab49fecbddd6e6af7

For details see https://github.com/QuantestPy/quantestpy/issues/129

Anyway, the skipping test problem that you have pointed out has been solved.